dozer-types/src/types/field.rs

```diff
@@ -42,6 +42,7 @@ pub enum FieldBorrow<'a> {
     Date(NaiveDate),
     Bson(&'a [u8]),
     Point(DozerPoint),
+    Array(Vec<Field>),
```

Comment:
Should be `&'a [Field]`

Reply:
it cannot be done that way because of #[derive(Deserialize)]

```
the trait bound `&'a [types::field::Field]: ingestion_types::_::_serde::Deserialize<'_>` is not satisfied [E0277] the trait `ingestion_types::_::_serde::Deserialize<'_>` is not implemented for `&'a [types::field::Field]`
```